docs/ENVIRONMENT.rst

```diff
@@ -8,6 +8,7 @@ It is possible to override some of the configuration parameters defined in the P
 Global/Universal
 ----------------
 -  **PATRONI\_CONFIGURATION**: it is possible to set the entire configuration for the Patroni via ``PATRONI_CONFIGURATION`` environment variable. In this case any other environment variables will not be considered!
+-  **PATRONI\_CONFIG\_FILE**: location of the configuration file. 
```

Comment:
Hi @daamien 

It looks like typo, the name should be `PATRONICTL\_CONFIG\_FILE`, and the better place for it is the 
[CTL](https://github.com/zalando/patroni/pull/1397/files#diff-1eb5a66e4f8c5c7a581dfee209c23d29R132) section.

Reply:
My bad ! I wrote this a bit too quickly. Here's a fix.

By the way, I noticed that the others PATRONI_CTL env variables in the  CTL section
( `PATRONI_CTL_INSECURE`, `PATRONI_CTL_CACERT`, etc. ) have inconsistant names ( PATRONICTL vs PATRONICTL) and theyr are absent in the code base. Maybe these variables are deprecated ?

Reply:
Yes, there is some little inconsistency, and no, these variables are not deprecated.

`PATRONICTL_CONFIG_FILE` describes where the config file is located.

Patronictl uses the normal Patroni config file, where one can define the [ctl](https://patroni.readthedocs.io/en/latest/SETTINGS.html#ctl) section.

On top of that, there is a way to override any of the configuration settings with ENV variables.
Most of the variable names are following the pattern `PATRONI_<SECTION>_<PARAMETER>`.
The name of the section is `ctl`, parameter names are `insecure`, 'cacert', and so on.